server/index.js

```diff
@@ -2,31 +2,28 @@ const connect = require('connect');
 const http = require('http');
 const serveStatic = require('serve-static')
 const path = require('path');
-const portfinder = require('portfinder');
 const setupProxy = require('../src/setupProxy');
+require('dotenv').config();
```

Comment:
should use `require('./src/env');` here as well

Reply:
Updated!